package.json

```diff
@@ -1,53 +1,69 @@
 {
-  "name": "leap-gov-util",
-  "version": "0.1.0",
-  "description": "LeapDAO governance utils",
-  "scripts": {
-    "test": "truffle compile && scripts/test.sh",
-    "clean": "rm -rf build/",
-    "compile": "truffle compile",
-    "deploy": "truffle deploy",
-    "test-ci": "npm run clean && npm run compile && scripts/test.sh",
-    "lint": "npm run lint:contracts && npm run lint:js",
-    "lint:contracts": "solium -d contracts/ --fix",
-    "lint:js": "eslint test/**/*.js --fix"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn lint"
-    }
-  },
-  "author": "LeapDAO",
-  "license": "MPL-2.0",
-  "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.2.1",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-register": "^6.26.0",
-    "bn.js": "^4.11.8",
-    "chai": "^4.1.2",
-    "chai-as-promised": "^7.1.1",
-    "dotenv": "^4.0.0",
-    "eslint": "^4.16.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-config-prettier": "^3.1.0",
-    "eslint-config-standard": "^11.0.0-beta.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-promise": "^3.6.0",
-    "eslint-plugin-standard": "^3.0.1",
-    "ethereumjs-util": "^5.1.3",
-    "ganache-cli": "^6.1.0",
-    "solium": "^1.1.8",
-    "truffle": "^4.1.13",
-    "truffle-hdwallet-provider": "0.0.5"
-  },
-  "dependencies": {
-    "cross-env": "^5.1.3",
-    "dotenv": "^4.0.0",
-    "openzeppelin-solidity": "1.12.0",
-    "pify": "^4.0.1",
-    "truffle-hdwallet-provider": "0.0.5"
-  }
+	"name": "LeapDAO-governance-utils",
```

Comment:
you seem to just copy-pasted the `package.json` from other project thus replacing all the things we had 🤯
Please use `package.json` already had and introduce just the changes you need for this PR

Reply:
@troggy ok. it's better to start new PR then